sdk/lib/core/map.dart

```diff
@@ -248,6 +285,26 @@ abstract class Map<K, V> {
   ///
   /// The operation is equivalent to doing `this[entry.key] = entry.value`
   /// for each [MapEntry] of the iterable.
+  /// ```dart
+  /// final map1 = {'A': 'X', 'B': 'Y', 'C': 'Z'};
+  /// map1.addEntries([const MapEntry('E', 'AB'), const MapEntry('F', 'CD')]);
+  /// print(map1); // {A: X, B: Y, C: Z, E: AB, F: CD}
```

Comment:
I just can't follow these letters easily, needs to be something else easier to follow